kaolin/io/modelnet.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2019-2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+from kaolin.io.dataset import KaolinDataset
+from kaolin.io.off import import_mesh
+
+import os
+
+class ModelNet(KaolinDataset):
+    r"""Dataset class for the ModelNet dataset.
+
+    Args:
+        root (str): Path to the base directory of the ModelNet dataset.
+        split (str): Split to load ('train' vs 'test', default: 'train').
+        categories (list): 
+            List of categories to load. If None is provided, 
+            all categories will be loaded. (default: None).
+    """
```

Comment:
as in https://github.com/NVIDIAGameWorks/kaolin/pull/375#discussion_r617842222
We should add a little description of returned value from `__getitem__`

Reply:
fixed.